settings.php

```diff
@@ -35,5 +35,78 @@
         0
     ));
 
-    // TODO: Options to configure default instance options.
+    // This section configures which contact methods should be displayed.
+    $settings->add(new admin_setting_heading('block_course_contacts/defaultheading',
+            get_string('method', 'block_course_contacts'), ''));
+
+    $yesno = [
+        0 => get_string('no'),
+        1 => get_string('yes')
+    ];
+    $settings->add(new admin_setting_configselect('block_course_contacts/email',
+            get_string('email', 'block_course_contacts'), '', 1, $yesno));
+
+    $settings->add(new admin_setting_configselect('block_course_contacts/message',
+            get_string('message', 'block_course_contacts'), '', 1, $yesno));
+
+    $settings->add(new admin_setting_configselect('block_course_contacts/phone',
+            get_string('phone', 'block_course_contacts'), '', 0, $yesno));
+
+    $settings->add(new admin_setting_configselect('block_course_contacts/description',
+            get_string('description', 'block_course_contacts'), '', 0, $yesno));
+
+    // This section configures which contact methods should be displayed for guest users.
+
+    $settings->add(new admin_setting_heading('block_course_contacts/methodguest',
+            get_string('methodguest', 'block_course_contacts'), ''));
+
+    $settings->add(new admin_setting_configselect('block_course_contacts/hide_block_guest',
+            get_string('hideblockforguest', 'block_course_contacts'), '', 1, $yesno));
+
+    $settings->add(new admin_setting_configselect('block_course_contacts/email_guest',
+            get_string('email', 'block_course_contacts'), '', 0, $yesno));
+
+    $settings->add(new admin_setting_configselect('block_course_contacts/message_guest',
+            get_string('message', 'block_course_contacts'), '', 0, $yesno));
+
+    $settings->add(new admin_setting_configselect('block_course_contacts/phone_guest',
+            get_string('phone', 'block_course_contacts'), '', 0, $yesno));
+
+    $settings->add(new admin_setting_configselect('block_course_contacts/description_guest',
+            get_string('description', 'block_course_contacts'), '', 0, $yesno));
+
+    // This section configures which contact methods should be displayed for guest users.
+    $settings->add(new admin_setting_heading('block_course_contacts/methodguest',
+            get_string('methodguest', 'block_course_contacts'), ''));
+
+    // This section gives options of how to display contacts.
+    $settings->add(new admin_setting_heading('block_course_contacts/displayheader',
+            get_string('display', 'block_course_contacts'), ''));
+
+    $sortby = [
+        0 => get_string('alphabetical', 'block_course_contacts'),
+        1 => get_string('recentlyactive', 'block_course_contacts'),
+        2 => get_string('dateenrolled', 'block_course_contacts')
```

Comment:
Same as previous: 

I don't think you need to key the numerical values like so. Is this some standard I'm not aware of?